simpleui-server/src/SuiZMQ.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * Created by Zack Beucler
+ * 
+ * potenitally add a send timeout ZMQ_SNDTIMEO
+ *                      http://api.zeromq.org/3-0:zmq-setsockopt
+ *      might be good to add because zmq will wait forever if not told otherwise
```

Comment:
In general, everything in production code should have a timeout. I struggle to think of an exception.

Reply:
I'll keep that in mind. Do you have a preference on how long the timeout should be? I was think it could be something large (like 30 seconds)

Reply:
For real-time-ish systems like this I generally favor timeouts on the order of 500 to 1000ms. Realistically, all of our operations should complete in a handful of milliseconds, so if something doesn't work instantly it's probably not gonna work ever. 

Another way to think about it: do you want to wait around for 30 seconds to learn if your button press succeeded or not